code/game/objects/items/charter.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/station_charter
```

Comment:
Iirc this is scannable by the mechanic scanner by default and you may need to check if it works even though you didn't set the build materials